seatunnel-api/src/main/java/org/apache/seatunnel/api/sink/SupportSchemaEvolutionSinkWriter.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.seatunnel.api.sink;
+
+import org.apache.seatunnel.api.table.schema.event.SchemaChangeEvent;
+
+import java.io.IOException;
+
+public interface SupportSchemaEvolutionSinkWriter {
```

Comment:
Should this interface extend SinkWriter to reduce the amount of interface for the implementing class.

Reply:
DDL should have an independent interface, and the connector can choose to support it, and use test cases to detect whether the user has implemented it correctly.

https://github.com/apache/seatunnel/pull/7463/files/4d062ebcfa9fabf37aff535280874dc1443cb30c#diff-4e38101d35f3d5901ed61ddd52530609bc8facb2d1e41476a981b18b3b2ffb9cR198


Other than that, I don't think of a better way. Using extends classes is more difficult to maintain.